query-engine/connectors/sql-query-connector/src/database/mod.rs

```diff
@@ -3,7 +3,12 @@ mod connection;
 mod js;
 mod transaction;
 
-#[cfg(feature = "native")]
+#[cfg(any(
+    feature = "mssql-native",
+    feature = "mysql-native",
+    feature = "postgresql-native",
+    feature = "sqlite-native"
+))]
```

Comment:
Why can't we alias it here?

Reply:
Aliases have to be set up per-crate and unless I am missing something this is a single usage of this combination in `sql-query-connectror`, so not worth it, IMHO.